File: pbwebcert/pbwebcert-jar/src/main/java/it/csi/pbandi/pbwebcert/integration/request/GestisciPropostaRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbwebcert.integration.request;

import it.csi.pbandi.pbwebcert.dto.FiltroRicercaProgettiNP;

public class GestisciPropostaRequest {
	private FiltroRicercaProgettiNP filtro;

	public FiltroRicercaProgettiNP getFiltro() {
		return filtro;
	}

	public void setFiltro(FiltroRicercaProgettiNP filtro) {
		this.filtro = filtro;
	}
}

File: pbwebcertwcl/src/app/gestione-proposte/commons/requests/gestisci-proposta-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { FiltroRicercaProgettiNP } from "../dto/filtro-ricerca-progetti-np";

export class GestisciPropostaRequest {
    constructor(
        public filtro: FiltroRicercaProgettiNP
    ) { }
}